hr_course/models/hr_course.py

```diff
@@ -180,6 +183,44 @@ def cancel_course(self):
         for record in self:
             record.write(record._cancel_course_values())
 
+    @api.model
+    def send_email(self, partner):
+        email = partner.email
+        if email and email.strip():
+            email_template = self.env.ref(
+                "hr_course.mail_template_validity_reminder_en"
+            )
+            options = {
+                "email_to": email,
+                "email_from": self.env["res.company"].search([]).email.strip(),
+            }
+            if email_template:
```

Comment:
```suggestion
```
don't need if condition, if the template will be removed it will raise on line 190